.github/workflows/cmd.yml

```diff
@@ -0,0 +1,417 @@
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            github.rest.reactions.createForIssueComment({
+              comment_id: ${{ github.event.comment.id }},
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              content: 'confused'
+            })
+
+      - name: Add 👍 reaction on success
+        uses: actions/github-script@v7
+        if: ${{ !failure() }}
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            github.rest.reactions.createForIssueComment({
+              comment_id: ${{ github.event.comment.id }},
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              content: '+1'
+            })
+
+  set-image:
+    needs: [ clean, is-org-member ]
+    if: ${{ startsWith(github.event.comment.body, '/cmd') && !contains(github.event.comment.body, '--help') && needs.is-org-member.outputs.member == 'true' }}
+    runs-on: ubuntu-latest
+    outputs:
+      IMAGE: ${{ steps.set-image.outputs.IMAGE }}
+      RUNNER: ${{ steps.set-image.outputs.RUNNER }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - id: set-image
+        run: |
+          BODY=$(echo "${{ github.event.comment.body }}" | xargs)
+          IMAGE_OVERRIDE=$(echo $BODY | grep -oe 'docker.io/paritytech/ci-unified:.*\s' | xargs)
+          
+          cat .github/env >> $GITHUB_OUTPUT
+          
+          if [ -n "$IMAGE_OVERRIDE" ]; then
+              echo "IMAGE=$IMAGE_OVERRIDE" >> $GITHUB_OUTPUT
+          fi
+          
+          if [[ $BODY == "/cmd bench"* ]]; then
+              echo "RUNNER=arc-runners-polkadot-sdk-benchmark" >> $GITHUB_OUTPUT
+          elif [[ $BODY == "/cmd update-ui"* ]]; then
+              echo "RUNNER=arc-runners-polkadot-sdk-beefy" >> $GITHUB_OUTPUT
+          else
+              echo "RUNNER=ubuntu-latest" >> $GITHUB_OUTPUT
+          fi
+
+  cmd:
+    needs: [ set-image ]
+    env:
+      JOB_NAME: 'cmd'
+    runs-on: ${{ needs.set-image.outputs.RUNNER }}
+    container:
+      image: ${{ needs.set-image.outputs.IMAGE }}
+    steps:
+      - name: Get command
+        uses: actions-ecosystem/action-regex-match@v2
+        id: get-pr-comment
+        with:
+          text: ${{ github.event.comment.body }}
+          regex: '^(\/cmd )([-\/\s\w.=:]+)$' # not ";" or && || to reduce risk of injections
+
+      - name: Build workflow link
+        if: ${{ !contains(github.event.comment.body, '--quiet') }}
+        id: build-link
+        run: |
+          # Get exactly the CMD job link, filtering out the other jobs
+          jobLink=$(curl -s \
+            -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+            -H "Accept: application/vnd.github.v3+json" \
+            https://api.github.com/repos/${{ github.repository }}/actions/runs/${{ github.run_id }}/jobs | jq '.jobs[] | select(.name | contains("${{ env.JOB_NAME }}")) | .html_url')
```

Comment:
Is this not predictable? I think i saw some other jobs predict their own CI run URL.

Reply:
in this case I'm building link specifically to `cmd` job, which is main source of information. Seemed better devex.